internal/providers/qemu/qemu.go

```diff
@@ -42,7 +37,19 @@ const (
 )
 
 func FetchConfig(f *resource.Fetcher) (types.Config, report.Report, error) {
-	f.Logger.Warning("Fetching the Ignition config via the Virtio block driver is currently experimental and subject to change.")
```

Comment:
Are we ready to mark this not-experimental?  Without some field experience, I'm not 100% convinced that _requiring_ an Ignition config is viable.